html/dpt3d.html

```diff
@@ -193,8 +193,8 @@
 			<div class="text">
 				<div id="guideBodyText" class="textList">
 					<p>Digital Peace Talks is a platform for conflicting views to discuss constructively. <br>This tour shows you how to use it.</p>
-					<p><img src="/tutorial_twofingers.png" alt="gestures" height="auto" width="100%"></p>
-					<p><img src="/tutorial_rightclick.png" alt="gestures" height="auto" width="100%"></p>
+					<!-- <p><img src="/tutorial_twofingers.png" alt="gestures" height="auto" width="100%"></p> -->
+					<!-- <p><img src="/tutorial_rightclick.png" alt="gestures" height="auto" width="100%"></p> -->
 					<p>Click on the futuristic text boxes to discuss the subject. <img src="/tutorial_click_topic.png" alt="click topic" height="auto" width="100%"></p>
 					<p>Click this symbol to add your opinion! <img src="/tutorial_opinions.png" alt="start opinion" height="auto" width="100%"></p>
```

Comment:
We need to use the edit icon (`<img src="/edit.png"...`) instead of `tutorial_opinions.png`.
Please verify that the edit icon sits good in the tutorial, as it has different dimensions than the other images in it.
If it doesn't fit well, let's talk